src/imageio/format/tiff.c

```diff
@@ -796,12 +794,11 @@ void gui_init(dt_imageio_module_format_t *self)
   dt_bauhaus_combobox_add(gui->compress, _("uncompressed"));
   dt_bauhaus_combobox_add(gui->compress, _("deflate"));
   dt_bauhaus_combobox_add(gui->compress, _("deflate with predictor"));
-  dt_bauhaus_combobox_add(gui->compress, _("deflate with predictor (float)"));
```

Comment:
this case has been removed, but I don't see any explanation.

Reply:
This case is redundant and confusing to the end user, there's really no need for it, and the functionallity has been merged into case 2; moreover, the current implementation can lead to an illegal TIFF writer configuration described in #6040